spec/spec_helper.rb

```diff
@@ -38,6 +34,10 @@ module SpecHelper
   config.raise_on_warning = true
 
   config.include(ExpectOffense)
+
+  config.include_context 'with default RSpec/Language config', :config
+  config.include_context 'config', :config
```

Comment:
After [this PR](https://github.com/rubocop/rubocop/pull/10806/files#diff-a328b4f030d5897fe22c01bb96a3bfbcc8bcc6a11f936d1e7a2db73a721d0eb8R52) is merged (and released, and we bump our `rubocop` dependency), this line becomes redundant.